Assets/Scripts/Menu/VolumeControl.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
```

Comment:
Unused references and audio control doesn't seem to carry over to the other scene